src/containers/RequestBuilder.js

```diff
@@ -78,7 +79,7 @@ export default class RequestBuilder extends Component {
   reconnectEhr() {
```

Comment:
Above this line (GitHub won't allow suggesting it), should this be changed due to the baseUrl and ehrUrl now being a part of the global state? The local state isn't used anywhere.

Reply:
yes